src/artifactory-client.ts

```diff
@@ -34,10 +34,15 @@ export interface AqlRequestResult<T> {
 
 export interface ArtifactoryClient {
 	query<T>(request: string): Promise<AqlRequestResult<T>>;
-	getContentStream(item: ArtifactoryItemMeta | string): Promise<stream.Readable>;
+	getContentStream(item: ArtifactoryItemMeta | string, byteRange?: ByteRange): Promise<IncomingMessage>;
 	resolveUri(item: ArtifactoryItemMeta | string): string;
 }
 
+export type ByteRange = {
```

Comment:
this isn't very important, but usually  `interface` is used for such situations. And I think its place is above interface `ArtifactoryClient`

Reply:
`ByteRange` is more like a tuple, not some interface to be implemented, so for me it is more natural to use `type`s in such cases)